ngraph/frontend/paddlepaddle/src/op/pool2d.cpp

```diff
@@ -66,7 +66,7 @@ namespace ngraph
                                         static_cast<uint64_t>(paddings[3])};
                         break;
                     default:
-                        throw std::runtime_error("Unsupported pooling paddings " + paddings.size());
+                        throw std::runtime_error("Unsupported pooling paddings " + std::to_string(paddings.size()));
```

Comment:
CMake Error at /home/runner/work/openvino/openvino/cmake/developer_package/clang_format/clang_format_check.cmake:15 (message):
  [clang-format] Code style check failed for:
  /home/runner/work/openvino/openvino/ngraph/frontend/paddlepaddle/src/op/pool2d.cpp

Reply:
fixed